build.sbt

```diff
@@ -466,6 +471,21 @@ lazy val `proxy-spanner` = (project in file("proxy/spanner"))
     assemblyMergeStrategy in assembly := {
       /*ADD CUSTOMIZATIONS HERE*/
       case PathList("META-INF", "io.netty.versions.properties") => MergeStrategy.last
+      case "module-info.class" => MergeStrategy.discard
+      // Perhaps can be removed later depending on how
+      // https://github.com/akka/akka-persistence-spanner/issues/75
+      // is resolved
+      case "GoogleInternetAuthorityG3.crt" => MergeStrategy.first
+      // These are the only 2 .proto with this problem because the version in
+      // cloudstate has deviated from the upstream one since
+      // https://github.com/cloudstateio/cloudstate/pull/127
+      // We should still not ship these .proto's in akka-persistence-spanner, though
+      case "google/api/http.proto" => MergeStrategy.first
+      case "google/api/annotations.proto" => MergeStrategy.first
+      // This one is different in akka-protobuf-v3.
+      // We should still not ship these .proto's in akka-persistence-spanner
+      // and not in akka-protobuf-v3, though.
+      case "google/protobuf/field_mask.proto" => MergeStrategy.first
```

Comment:
I think you can safely discard all ".proto"-files since the fat jar will not have them used for anything, it's only intended to be executed via java -jar